Sources/AWSLambdaRuntimeCore/NewLambdaRuntime.swift

```diff
@@ -34,5 +34,35 @@ package final class NewLambdaRuntime<Handler>: Sendable where Handler: Streaming
     }
 
     package func run() async throws {
+        guard let runtimeEndpoint = Lambda.env("AWS_LAMBDA_RUNTIME_API") else {
+            throw NewLambdaRuntimeError(code: .cannotStartLambdaRuntime)
+        }
+
+        let ipAndPort = runtimeEndpoint.split(separator: ":", maxSplits: 1)
+        let ip = String(ipAndPort[0])
+        let port = Int(ipAndPort[1])!
```

Comment:
we should throw an error here, if the second part is not an Int. `.invalidPort`.